examples/starlog/src/components/BaseHead.astro

```diff
@@ -1,5 +1,5 @@
 ---
-import { ViewTransitions } from 'astro:transitions';
+import { ClientRouter } from 'astro:transitions';
```

Comment:
Is there at all a concern regarding the fact that you still import this from `astro:transitions`? Perhaps it should be exported from `astro/components` now as well and documented as such?

Reply:
Because it's "not real view transitions"? It still uses view transitions; that's still the biggest part of the router. It's not like the other stuff inside `astro:transitions` are not also enhancements to view transitions. This is no different. It's just a rename to clarify that it's an enhancement.

Reply:
I meant more so in a marketing sense. If we start selling this to users outside of the context of view transitions (e.g. "If you need a client side router, use this") it could be confusing why it's imported from `astro:transitions`

Reply:
It's still pretty deeply tied to view transitions. The `fallback` property is about view transitions. The animations that occur are view transitions. If you wanted to disable the animations you need to do `transition:animate="none"`. It's not a traditional SPA style router and I don't think we should rethink everything just for a rename.